src/api/python/z3/z3util.py

```diff
@@ -446,9 +446,9 @@ def myBinOp(op, *L):
         L = L[0]
 
     if z3_debug():
-        assert all(not isinstance(l, bool) for l in L)
```

Comment:
```
z3/z3util.py:449:48: E741 ambiguous variable name 'l'
```

And it is actually confusing, I had hard times trying to visually find where is `l` and where is `L`